app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -0,0 +1,154 @@
+    </p>
+  </section>
+
+  <%= render SubjectSpecific::TextBlockWithTwoImagesComponent.new(
+    heading: "Bring it",
+    colour: "pink",
+    image_1_path: "media/images/content/blog/abigail-beeley.jpg",
+    image_1_alt: "Maths teacher Abigail Beeley in front of a maths display board.",
+    image_2_path: "media/images/content/blog/giorgio-rubbo.jpg",
+    image_2_alt: "Biology teacher Giorgio Rubbo wearing a suit and tie.",
+  ) do %>
+
+    <p>
+      Every teacher brings something different to the classroom.
+    </p>
+
+    <p>
+    What could you bring?
+    </p>
+  <% end %>
+
+  <section>
+    <figure>
+      <blockquote class="pink-quotes">
+        <p class="bold">
+          I promise the level of satisfaction is like nothing you’ll get in another job. I wake up every morning and want to get into school.
+        </p>
+      </blockquote>
+
+      <figcaption>Phil, teacher</figcaption>
+    </figure>
+  </section>
+
+  <section class="columns">
+    <div>
+      <h2 class="heading-s">What explore teaching advisers can do</h2>
+
+      <p>
+        Your adviser can help you:
+      </p>
+    </div>
+
+    <div>
+      <ul class="badges">
+        <li class="badge">understand what teaching is really like</li>
+        <li class="badge">connect with other students</li>
+        <li class="badge">build your confidence and transferable skills</li>
+        <li class="badge">get classroom experience</li>
+        <li class="badge">understand the process to become a teacher</li>
+      </ul>
+    </div>
+  </section>
+
+  <section>
+    <h2 class="heading-s">How to get an explore teaching adviser</h2>
+
+    <p>
+        To be eligible for an explore teaching adviser, you just need:
+    </p>
+
+    <ul>
+      <li>
+        to be studying for an undergraduate degree, not in your final year of study
+      </li>
+      <li>
+        an interest in teaching
+      </li>
+    </ul>
+
+    <p>
+    If you're in your final year of study or you already have a degree, you can still get a teacher training adviser.
+    </p>
+
+    <p>
+      <a href="/teacher-training-advisers">Find out how a teacher training adviser could help you</span></a>.
+    </p>
+
+    <p class="call-to-action__action">
+      <a href="https://adviser-getintoteaching.education.gov.uk/?channel=222750047">Get an <span>explore teaching adviser</span></a>
+    </p>
+  </section>
+
+  <%= render SubjectSpecific::TextBlockWithOneImageComponent.new(
+    heading: "Is teaching right for me?",
+    colour: "yellow",
+    image_path: "media/images/content/hero-images/0011.jpg",
+    image_alt: "Child pointing something out to a teacher in the classroom.",
+  ) %>
+
+  <section>
+    <h2 class="heading-s">Use your subject passion</h2>
+
+    <p>
+    Bring your subject to life for your pupils. Help them be inspired by it just like you.
```

Comment:
This sounds a little clunky - are the pupils inspired by me or am I inspired by the subject?

```suggestion
    Bring your subject to life for your pupils. Help them be inspired by it just like you are.
```
or 'in the same way you are' or 'Help them be as inspired by it as you are' or similar!